index.html

```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
+      <li class="about-li">Back End Dev</li>
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
+
+  <section class="third-wrapper">
+    <div class="image-wrapper">
+      <img class="snapshoot" src="./assets/third-wrapper.png">
+    </div>
+   
+    <h2 class="poppins">Multi-post <br/>Stories</h2>
+    <ul class="poppins" id="about-ul">
+      <li class="about-li" id="canopy">CANOPY</li>
+      <li class="about-li">Back End Dev</li>
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
+
+  <section class="forth-wrapper">
+    <div class="image-wrapper">
+      <img class="snapshoot" src="./assets/forth-wrapper.png">
+    </div>
+   
+    <h2 class="poppins">Tonic</h2>
+    <ul class="poppins" id="about-ul">
+      <li class="about-li" id="canopy">CANOPY</li>
+      <li class="about-li">Back End Dev</li>
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
+
+  <section class="fifth-wrapper">
+    <div class="image-wrapper">
+      <img class="snapshoot" src="./assets/fifth-wrapper.png">
+    </div>
+   
+    <h2 class="poppins">Multi-post <br/>Stories</h2>
+    <ul class="poppins" id="about-ul">
+      <li class="about-li" id="canopy">CANOPY</li>
+      <li class="about-li">Back End Dev</li>
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
+
+  <section class = "sixth-wrapper">
+
+    <h1 class="poppins">About<br/>Myself</h1>
```

Comment:
- [ ] kindly make sure that your `About Myself ` title has the correct colour. 
`color: #172B4D;`